packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts

```diff
@@ -251,7 +264,7 @@ export async function importNodesToEnvironment(environmentRegistryEntry: FabricE
         const newEnvironment: FabricEnvironment = new FabricEnvironment(environmentRegistryEntry.name);
         const newNodes: FabricNode[] = await newEnvironment.getNodes();
         if (newNodes.length === 0) {
-            await fs.remove(environmentPath);
+            await fs.remove(environmentNodesPath);
```

Comment:
in the case of a non ops tools one this needs to remove the environment

Reply:
Indeed, the behavior is now different from what is currently expected, I'll change that.